BrowserKit/Sources/WebEngine/WKWebview/Scripts/WKUserScriptManager.swift

```diff
@@ -7,9 +7,10 @@ import Foundation
 
 /// Manager used to inject scripts at document start or end inside a WKEngineWebView
 protocol WKUserScriptManager {
-    func injectUserScriptsIntoWebView(_ webView: WKEngineWebView)
+    func injectUserScriptsIntoWebView(_ webView: WKEngineWebView) async
 }
 
+@MainActor
```

Comment:
This was needed since we call `injectUserScripts` in the init which calls `WKUserScript()` further down, and it would crash if its not on the main thread